demo/EV3D4/README.md

```diff
@@ -1,21 +1,18 @@
 # EV3D4
 EV3D4 is designed to look like R2-D2 from Star Wars. There are two options for
 controlling EV3D4. The first is to use the IR remote to send commands to the IR
-sensor, run EV3D4RemoteControl to use this method. The second means of
-controlling EV3D4 is via a web browser, run EV3D4WebControl to use this method.
+sensor, run EV3D4RemoteControl.py to use this method. The second means of
```

Comment:
Can we get a quick explanation of how to "run <file>"? Either instructions to `chmod +x <file>` and run from Brickman or to do it from the command line.